regex-assembly/942380.ra

```diff
@@ -3,23 +3,19 @@
 
 ##!+ i
 
-\bhaving\b\s+\d{1,10}\s*?[=<>]
-\bhaving\b\s+'[^=]{1,10}'\s*?[=<>]
-\bexecute\(
-\bexecute\s{1,5}[\w\.$]{1,5}\s{0,3}
-\bhaving\b ?\d{1,10} ?[=<>]+
-\bhaving\b ?[\'\"][^=]{1,10}[\'\" ?[=<>]+
-\bcreate\s+?table.{0,20}?\(
+\bhaving\s*\(?\s*(?:['"]?\s*\d+\s*['"]?|['"]\s*[^\d]*\s*['"]|true|false)\s*(?:[=<>+\-*/%^&!|]+|div|mod)
+\bexecute\s*\(
+\bexecute\b\s*`?\s*[\w\.$]+\s*`?
+\bcreate\b\s+\btable\b\s*`?\s*.+?\s*`?\s*\(
```

Comment:
The `\b`s are redundant since `\s+` implies non-word characters between the two strings.
```suggestion
\bcreate\s+table\b\s*`?\s*.+?\s*`?\s*\(
```